test/python/WMCore_t/MicroService_t/Tools_t/Common_t.py

```diff
@@ -31,36 +35,44 @@ def testDbsInfo(self):
         self.assertEqual(expect, sizes)
         self.assertEqual(len(self.datasets), len(datasetTransfers))
 
+    @attr("integration")
     def testGetEventsLumis(self):
         "Test function for getEventsLumis()"
         totEvts = totLumis = 0
         for dataset in self.datasets:
             nevts, nlumis = getEventsLumis(dataset, self.dbsUrl)
+            print("dataset: {} with evts: {} and lumis: {}".format(dataset, nevts, nlumis))
             totEvts += nevts
             totLumis += nlumis
-        expect = 10250 + 10616
+        expect = 7398 + 0
         self.assertEqual(expect, totEvts)
-        expect = 22 + 10
+        expect = 16 + 0
         self.assertEqual(expect, totLumis)
 
+    @attr("integration")
     def test_findParent(self):
         "Test function for findParent()"
         parents = findParent(self.child, self.dbsUrl)
         self.assertEqual(parents[self.child[0]], '/SingleElectron/Run2016B-v2/RAW')
 
-    def test_isEmptyResults(self):
-        row = {'code': 200, 'data': None}
-        result = isEmptyResults(row)
-        self.assertEqual(result, True)
-        row = {'code': 200, 'data': []}
-        result = isEmptyResults(row)
-        self.assertEqual(result, True)
-        row = {'code': 400, 'data': None}
-        result = isEmptyResults(row)
-        self.assertEqual(result, False)
-        row = {'code': 400, 'data': []}
-        result = isEmptyResults(row)
-        self.assertEqual(result, False)
+    def testHasHTTPFailed(self):
+        """Test the hasHTTPFailed method"""
+        self.assertTrue(hasHTTPFailed({}))
+        self.assertTrue(hasHTTPFailed({'data': 1, 'code': 400, 'error': 'blah'}))
```

Comment:
please add additional test with
```
{''data': '', 'code': 301}
```
and handle it correctly according how code will be adjusted.